### PR TITLE
pass err through

### PR DESCRIPTION
**Pull request by mvayngrib**
i forgot to pass the err through

### CODE REVIEW

File: lib/dns.js [hunk 1]
```diff
@@ -153,7 +153,9 @@ Client.prototype.destroy = function() {
     var cb = args[args.length - 1];
     args[args.length - 1] = function(err, result) {
       client.destroy();
-      cb(null, result);
+      
+      if (err) cb(err);
```
Comment (indutny) [inline lib/dns.js]: Let's just do it in a following way: `cb(err, result)`. It is quite similar, IMHO.

**Comment by indutny** [comment]
One minor nit, otherwise LGTM! Thank you!

**Comment by mvayngrib** [comment]
sounds good, updated PR

**Comment by indutny** [comment]
Landed and released in 1.0.1, thank you!

**Pull request closed without merging**